src/golang/cmd/server/handler/list_integrations.go

```diff
@@ -77,8 +78,11 @@ func (h *ListIntegrationsHandler) Perform(ctx context.Context, interfaceArgs int
 
 	responses := make([]integrationResponse, 0, len(integrations))
 	for _, integrationObject := range integrations {
-		response := convertIntegrationObjectToResponse(&integrationObject)
-		responses = append(responses, *response)
+		state, err := execution_state.ExtractConnectionState(&integrationObject)
```

Comment:
What's the motivation of this change? It seems that we are only showing succeeded integrations here. However, should we show them as failure / in progress accordingly?

Reply:
I also don't quite understand why your demo works and does show an in-progress connection (lambda1) in the integration list.

Reply:
I have tested it and that's my bad. My initial thought is to block user from seeing and using integration that has not completed. However, the UI will also get affected and do not display as expected

Reply:
I will make changes upon that.

Reply:
Sounds good, thanks!